flepimop/R_packages/inference/R/inference_to_forecast.R

```diff
@@ -28,6 +28,7 @@ cum_death_forecast <- function (sim_data,
   
 }
 
+
```

Comment:
?

Reply:
> Looks good; one weird space diff, but that's likely test cruft?

Yeah, exactly. Just wanted to demonstrate what it would look like, but not needed and reverted.